app/Http/Controllers/PhotoController.php

```diff
@@ -287,6 +287,9 @@ public function setLicense(SetPhotoLicenseRequest $request): void
 	public function setUploadDate(SetPhotoUploadDate $request): void
 	{
 		$request->photo()->created_at = $request->requestDate();
+		if ($request->photo()->updated_at->isBefore($request->requestDate())) {
```

Comment:
This is unnecessary. When doing `->save()` on a model, laravel will update the value of `updated_at` to the timestamp of the operation.